ENH: Clean up docker file

jhlegarreta opened the pull request:
- STYLE: Remove trailing whitespace and add newline at EOF
- ENH: Remove redundant directory change statement

Resolves #6.

jhlegarreta committed:
STYLE: Remove trailing whitespace and add newline at EOF

Remove trailing whitespace and add newline at EOF.

jhlegarreta committed:
ENH: Remove redundant directory change statement

Remove redundant directory change statement as there is another one
changing to the same directory immediately before.

jhlegarreta commented on Dockerfile:
```diff
@@ -41,15 +41,14 @@ RUN rm -rf /var/lib/apt/lists/* && \
         | tar -xz -C /opt/ants-2.3.4 --strip-components 1
 
 RUN pip install -U pip
```
Is this necessary?

felixdumais replied:
Nope, you can remove

jhlegarreta replied:
Done in [6888d13](https://github.com/scil-vital/dockerfile_tractolearn/pull/7/commits/6888d13dd817844f74983c5d0115dd5075b4e1f2).

jhlegarreta commented on Dockerfile:
```diff
@@ -41,15 +41,14 @@ RUN rm -rf /var/lib/apt/lists/* && \
 
 RUN pip install -U pip
-RUN cd /tractolearn && pip install -e . 
+RUN cd /tractolearn && pip install -e .
```
We are using here a bare `pip` whereas all other commands use the Python 3.8 version `pip`. I would use `pip` everywhere to save typing and make maintenance easier. As we are installing a single Python version, `pip` should be able to resolve to that verstion.

felixdumais replied:
Yep, I tested it on my side, you can remove the pip3.8

jhlegarreta replied:
Done in [268d578](https://github.com/scil-vital/dockerfile_tractolearn/pull/7/commits/268d578fa57f899c826f542f9055a68583682063).

jhlegarreta committed:
STYLE: Do not specify a Python version in `pip` command

Do not specify a Python version in `pip` command: since a single Python
version is installed, a single `pip` version is installed so there is no
need to specify the version.

jhlegarreta committed:
ENH: Remove unnecessary `pip install` command

Remove unnecessary `pip install` command: `pip` gets installed and
upgraded a few statements earlier.
